CMakeLists.txt

```diff
@@ -17,16 +13,21 @@ else()
 endif()
 
 find_package(PythonExtensions REQUIRED)
-find_package(Python COMPONENTS Interpreter Development)
+if(CMAKE_VERSION VERSION_LESS 3.18)
+    find_package(Python COMPONENTS Interpreter Development)
+else()
+    find_package(Python COMPONENTS Interpreter Development.Module)
```

Comment:
I don't really understand why this worked before on manylinux (`Development.Embed` is missing there), but might as well be specific. :)

Reply:
It works because it is not set as required I think. So it simply fails to find it, but still continues forward

Reply:
Ahh, you are very likely right. Should it be set as required, though?

Reply:
Yes it probably should be marked as required